x/stake/stake_migration/pub.go

```diff
@@ -0,0 +1,30 @@
+package stake_migration
+
+import (
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/pubsub"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/stake/cross_stake"
+)
+
+func PublishStakeMigrationEvent(ctx sdk.Context, keeper cross_stake.Keeper, from string, to []pubsub.CrossReceiver, symbol string,
+	eventType string, relayerFee int64,
+) {
+	if keeper.PbsbServer != nil {
```

Comment:
No need this check, for we have this check when calling this function.

Reply:
Done